pkg/apply/types.go

```diff
@@ -57,12 +57,7 @@ type UpdateChangesTracker struct {
 	Topic  string     `json:"topic"`
 
 	// tracks changes in partition count
-	// TODO: implement this
-	// NumPartitions *IntValueChanges `json:"numPartitions"`
-
-	// tracks changes in replication factor
-	// TODO: implement this
-	// ReplicationFactor *IntValueChanges `json:"replicationFactor"`
```

Comment:
`ReplicationFactor` was deleted because topicctl doesn't support increasing a topic's replication factor beyond the highest ISR of a partition of that topic: https://github.com/getsentry/topicctl/blob/master/pkg/apply/apply.go#L514-L529